frontend/src/modules/integration/components/integration-list-item.vue

```diff
@@ -1,136 +1,97 @@
-      <app-integration-connect :integration="integration">
-        <template
-          #default="{
-            connect,
-            connected,
-            settings,
-            hasSettings,
-            settingsComponent,
-            hasIntegration,
-          }"
-        >
-          <div class="flex items-center justify-between">
-            <el-button
-              v-if="!connected"
-              class="btn btn--secondary btn--md"
-              @click="connect"
-            >
-              {{
-                (integration.premium || integration.scale) && !hasIntegration
-                  ? "Upgrade Plan"
-                  : "Connect"
-              }}
-            </el-button>
-            <el-button
-              v-else
-              class="btn btn--secondary !text-red-500 btn--md"
-              :loading="loadingDisconnect"
-              @click="handleDisconnect"
-            >
-              Disconnect
-            </el-button>
-            <component :is="settingsComponent" v-if="connected && hasSettings && settingsComponent" :integration="integration" />
-            <el-button
-              v-else-if="connected && hasSettings"
-              class="btn btn-link btn-link--md btn-link--primary"
-              @click="settings"
+    <div class="flex-grow" />
+
+    <app-integration-connect :integration="integration">
+      <template
+        #default="{
+          connect,
+          connected,
+          settings,
+          hasSettings,
+          settingsComponent,
+        }"
+      >
+        <!-- Connect -->
+        <div v-if="!connected" class="px-5 pb-5">
+          <cr-button type="secondary" class="w-full" @click="connect">
+            <i class="ri-link" /> Connect
+          </cr-button>
+        </div>
+        <div v-else-if="isInProgress" class="bg-gray-50 py-3 px-5 min-h-14">
+          <app-integration-progress-bar :progress="selectedProgress" />
+        </div>
+        <div v-else class="bg-gray-50 py-3 px-5 min-h-14 flex items-center justify-between">
+          <div>
+            <component :is="settingsComponent" v-if="connected && settingsComponent" :integration="integration" />
+            <el-tooltip
+              v-if="isDone"
+              :content="lastSynced.absolute"
+              placement="top"
             >
-              <i class="ri-settings-2-line mr-2" />Settings
-            </el-button>
+              <p class="text-3xs text-gray-500">
+                {{ lastSynced.relative }}
+              </p>
+            </el-tooltip>
           </div>
-        </template>
-      </app-integration-connect>
-    </div>
+          <div>
+            <el-dropdown placement="bottom-end">
+              <cr-button size="small" type="tertiary-light-gray" :icon-only="true">
+                <i class="ri-more-fill" />
+              </cr-button>
+              <template #dropdown>
+                <el-dropdown-item v-if="hasSettings" @click="settings">
+                  Settings
```

Comment:
This one is missing the Cog icon and it should be "Integration Settings" instead of "Settings"